python/dgl/transform.py

```diff
@@ -1255,56 +1255,139 @@ def out_subgraph(g, nodes):
         ret.edges[etype].data[EID] = induced_edges[i].tousertensor()
     return ret
 
-def to_simple(g, return_counts='count', writeback_mapping=None):
-    """Convert a heterogeneous multigraph to a heterogeneous simple graph, coalescing
-    duplicate edges into one.
+def to_simple(g, return_counts='count', writeback_mapping=True, share_ndata=True):
```

Comment:
We can still have the option of share_edata. If True, we will randomly use one of the features for duplicate edges.

Reply:
> We can still have the option of share_edata. If True, we will randomly use one of the features for duplicate edges.

I don't like this design. This only introduces the uncertainty.

Reply:
We can set this argument to False by default. Meanwhile, this can be a natural behavior if the duplicate edges will have the same feature. If I am a user who wants to copy the edge features from the original graph to the converted simple graph, then I need to first compute a mapping from the edges in the simple graph to the edges in the original graph, which can be tedious.